code/modules/modular_computers/computers/item/tablet.dm

```diff
@@ -42,9 +42,74 @@
 	has_variants = FALSE
 	device_theme = "syndicate"
 
+/obj/item/modular_computer/tablet/nukeops/Initialize()
+	. = ..()
+	set_light_color(COLOR_RED) //Syndicate likes it red
```

Comment:
Just set it on the variable definition? Why would you need to do this on `Initialize()`?